src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -421,6 +422,22 @@ public string SelectedPackagePathForInstall
         /// Show Run Preview flag.
         /// </summary>
         public bool ShowRunPreview { get; set; }
+
+        
+        private ObservableCollection<RequiredProperty> requiredProperties;
+        /// <summary>
+        /// Collection of RequiredProperties, as set in the DynamoSettings file
+        /// </summary>
+        public ObservableCollection<RequiredProperty> RequiredProperties
+        {
+            get => requiredProperties;
+            set
+            {
+                requiredProperties = value;
+                RaisePropertyChanged(nameof(RequiredProperties));
```

Comment:
you generally shouldn't need to raise property changes for observable collections...